Tests/test_content.py

```diff
@@ -34,8 +40,7 @@
 
 def options_handler():
     parser = argparse.ArgumentParser(description='Utility for batch action on incidents')
-    parser.add_argument('-u', '--user', help='The username for the login', required=True)
-    parser.add_argument('-p', '--password', help='The password for the login', required=True)
+    parser.add_argument('-k', '--apiKey', help='The Demisto API key for the server', required=True)
     parser.add_argument('-s', '--server', help='The server URL to connect to')
```

Comment:
Let's remove this server param

Reply:
i think that it is used for the non_ami ones